File: Hache.Server/Entities/Articulo.cs
namespace Hache.Server.Entities
{
    public class Articulo
    {
        public int ID_Articulo { get; set; }
        public string Nombre { get; set; } = string.Empty;
        public decimal Precio { get; set; }
        public Categoria Categoria { get; set; } = new Categoria();
        public Marca Marca { get; set; } = new Marca();
        public List<Imagen> Imagen{ get; set; } = new List<Imagen>();

        public Articulo() { }
    }
}

File: hache.client/src/app/core/models/articulo.ts
import { Categoria } from "./categoria";
import { Imagen } from "./imagen";
import { Marca } from "./marca";

export class Articulo {

  id!: number;
  nombre!: string;
  precio!: number;
  categoria!: Categoria;
  marca!: Marca;
  imagen!: Imagen[];
  codigoXubio?: string;
  cantidad?: number;

  //Implementados en siderbar de inventario

  cantidadActual?: number;
  idStock?: number;
}
